README.md

```diff
@@ -44,6 +44,29 @@ custom:
     - SomeOtherKeyThatAssignsRandomValue
 ```
 
+### Specifying usage plan (To put both keys in the same usage plan)
+
+```yaml
+custom:
```

Comment:
@christopherchubb - i the code i see provision for `defaultUsagePlan`.. can you add a sample to `README.md`

Reply:
Ah, good catch. Thanks.